### PR TITLE
RDM-13015 FTs for the new CaseAccessCategory field in the Global Search endpoint

### PR DESCRIPTION
**Pull request by marianadpereira**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-13015

### Change description ###

FTs for the new CaseAccessCategory field in the Global Search endpoint.

This is also enabling F-1005.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
ES tests are skipped in the successful build report. We need to see all FTs passing as otherwise some may fail in master build again.

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -19,7 +19,7 @@ def branchesToSync = ['demo', 'ithc', 'perftest']
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-1174"  // This doesn't change frequently, but when it does, only change this value.
```
Comment (marianadpereira) [inline Jenkinsfile_CNP:22]: This needs to be reverted before merging.

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -94,7 +94,7 @@ env.CASE_DOCUMENT_AM_URL = "http://ccd-case-document-am-api-aat.service.core-com
 env.RD_LOCATION_REF_API_BASE_URL = "http://rd-location-ref-api-aat.service.core-compute-aat.internal"
 env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
-env.ELASTIC_SEARCH_FTA_ENABLED = "false"
+env.ELASTIC_SEARCH_FTA_ENABLED = "true"
```
Comment (marianadpereira) [inline Jenkinsfile_CNP:97]: This needs to be reverted before merging.

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 3]
```diff
@@ -27,7 +27,7 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1174.service.core-compute-preview.internal/
```
Comment (marianadpereira) [inline charts/ccd-data-store-api/values.preview.template.yaml:30]: This needs to be reverted before merging.

**Review by MSancaktutar — APPROVED** [review]
lgtm

**Review by MSancaktutar — APPROVED** [review]

**Pull request merged**